docs/docs/reference/olap-engines/duckdb.md

```diff
@@ -25,4 +25,6 @@ Well now you can! For more details, see our page on [Using Multiple OLAP Engines
 
 ## Additional Notes
 
-- For dashboards powered by DuckDB, [measure definitions](/build/dashboards/dashboards.md#measures) are required to follow standard [DuckDB SQL](https://duckdb.org/docs/sql/introduction) syntax.
\ No newline at end of file
+- For dashboards powered by DuckDB, [measure definitions](/build/dashboards/dashboards.md#measures) are required to follow standard [DuckDB SQL](https://duckdb.org/docs/sql/introduction) syntax.
+- Known issue around creating a DuckDB via the UI, you will need to create the YAML file manually.
+- Note that local running instances of DuckDB (outside of the default one) is mainly for development and local use. When uploading to Rill Cloud, it is recommended to use a `source` type instead
```

Comment:
Should the above two points go into this page or the "MotherDuck / DuckDB" connector page since this is more about DuckDB as a source vs an OLAP engine (perhaps we may need to further think about how we organize connectors vs OLAP engine docs)?

Reply:
Same as the previous comment, DuckDB should only be added as a source even though its an OLAP engine but then how do we make this less confusing for our users...